batavia/core/exceptions.js

```diff
@@ -22,16 +22,61 @@ BaseException.prototype.toString = function() {
 }
 
 BaseException.prototype.__str__ = function() {
-    if (this.msg) {
-        return this.msg
+    if (this.args !== undefined) {
+        if (this.args.length == 1) {
+            return this.args[0].toString()
+        }
+        // Multiple args. Format like a tuple.
+        let result = '(' + arg.toString()
+        for (arg in this.args.slice(1)) {
+            result += ', ' + arg.toString()
+        }
+        return result = ')'
```

Comment:
This should be result + ')'.

That's why `StopIteration.__str__()` is undefined.